distributed/worker.py

```diff
@@ -1809,6 +1809,10 @@ def release_key(self, key, cause=None, reason=None, report=True):
                 self.batched_stream.send({'op': 'release',
                                           'key': key,
                                           'cause': cause})
+            self._throttledGC.collect(
+                force_gc=True if nbytes_to_free > 10 * 2**20 else False
```

Comment:
Could probably drop the ternary statement and just replace with the condition.

    force_gc = nbytes_to_free > 10 * 2**20